src/usdb_syncer/gui/song_table/song_table.py

```diff
@@ -149,20 +149,23 @@ def _setup_view(self) -> None:
                 existing_state = True
                 self._search.order = Column(header.sortIndicatorSection()).song_order()
                 self._search.descending = bool(header.sortIndicatorOrder().value)
+
+        header.setSectionsMovable(True)
+        header.setSectionsClickable(True)
```

Comment:
What does this do?

Reply:
I thought it was required in addition to `setSectionsMovable()`, but it isn't.